examples/load_balancer/load_balancer.c

```diff
@@ -302,60 +334,48 @@ print_flow_info(struct flow_info *f) {
 }
 
 /*
- * Parse and assign load balancer server/client interface information
+ * Parse and print load balancer server/client interface information, check port validity
  */
 static void
-get_iface_inf(void) {
-        int fd, i;
-        struct ifreq ifr;
+print_iface_inf(void) {
```

Comment:
Rename this to something like validate_configuration because thats the main thing its doing, print sounds a bit misleading

Reply:
Adjusted